projects/storefrontstyles/scss/components/cart/_cart-proceed-to-checkout.scss

```diff
@@ -0,0 +1,16 @@
+%cx-cart-proceed-to-checkout {
```

Comment:
since the styling is not available here, but inherits from the progress button styles, 

did the UI/UX team ask to make the text bold? Did they respond regarding the look and feel of thing
![Screen Shot 2022-01-23 at 11 19 03 PM](https://user-images.githubusercontent.com/29099466/150721525-4cfb2d66-da63-4500-9148-93b241fd7a39.png)
![Screen Shot 2022-01-23 at 11 19 21 PM](https://user-images.githubusercontent.com/29099466/150721528-f66fe95a-c501-4530-8a98-6b9a88fcb68a.png)
s? My ping in this PR has no response regarding this matter

Reply:
The progress button was styled followed by a style guide for this button. So probably in the Cart, the old button has old styles from the past design style guides.

Reply:
@bgambocjaviniar yes, UI/UX approved, this is an accessibility update, bold was added since contrast/font-size was not passing the validator. More details on GH-13498